omnibus/files/private-chef-cookbooks/private-chef/providers/pg_sqitch.rb

```diff
@@ -28,7 +28,8 @@
                deploy #{target} --verify
       EOM
       environment "PERL5LIB" => "", # force us to use omnibus perl
-                  "PGPASSWORD" => new_resource.password
+                  "PGPASSWORD" => new_resource.password,
+                  "PGSSLMODE" => node['private_chef']['postgresql']['sslmode']
```

Comment:
sslmode should really be added as a property on the resource and the resource should be called with the node object rather than directly using the node object from the provider like this.

Reply:
Thanks, good catch!